packages/esm-patient-vitals-app/translations/en.json

```diff
@@ -13,6 +13,8 @@
   "checkForValidity": "Some of the values entered are invalid",
   "date": "Date",
   "dateAndTime": "Date and time",
+  "daysOldVitals_one": "These vitals are {{count}} day old",
+  "daysOldVitals_other": "These vitals are {{count}} days old",
```

Comment:
@miirochristopher we don't update these by hand. Rather, when you add new translation strings in your code, i18next-parser will automatically extract them from your code when you commit your changes. Updates to non-en locale files are handled automatically via the Transifex integration. 

tl;dr, don't update translation files manually.